PT-2211 This pstack return error code line number

baotiao opened the pull request:
[PT-2211 This pstack return error code line number](https://github.com/percona/percona-toolkit/commit/b6626a5784b746e8e133ba2330488d73b9fd7066)

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/percona-toolkit?pullRequest=628) <br/>All committers have signed the CLA.

baotiao committed:
PT-2211 This pstack return error code line number

svetasmirnova reviewed (changes requested):
Could you please add a test case? You will find examples in the contribution guide and in the directory t/pt-pmp

svetasmirnova committed:
PT-2211 - This pstack return error code line number

- Added test case sample for this case
- Adjusted sample results for t/pt-pmp/samples/stacktrace003.in that originally
  returned wrong data

svetasmirnova commented:
OK, added test samples myself.
